submissions/akulahere/task-js-post-oop/index.js

```diff
@@ -0,0 +1,65 @@
+class Inhabitant {
+  constructor(species, name, gender, legs, hands, saying) {
+    this.species = species;
+    this.name = name;
+    this.gender = gender;
+    this.legs = legs;
+    this.hands = hands;
+    this.saying = saying;
+    this.friends = Array.isArray(this.friends) ? this.friends : 'No friends' ;
```

Comment:
At object construction time `this.friends` is not defined, so `Array.isArray(this.friends)` will always be `false`.
Is there a good reason to translate list of items into string at this stage?

Reply:
fixed